test/test_span.cpp

```diff
@@ -564,19 +564,17 @@ TEST_CASE("span portability", "[PortsOfCall::span]") {
   constexpr const std::size_t Nb = N * sizeof(Real);
   constexpr const Real tol = 1.0E-8;
 
-  constexpr const Real pi = std::acos(-1);
+  constexpr const Real pi = 3.141592653589793238462643383279502884;
```

Comment:
Why not just use `M_PI`?

Reply:
Paranoid now about portability in the libraries. If you prefer `M_PI` let me know.

Reply:
All things being equal, I prefer `M_PI`, which is a macro set to that decimal isn't it? But if you think there's reasons to be paranoid, let's talk.